app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -370,6 +380,7 @@ public void onClick(View v) {
             @Override
             public void onClick(View v) {
                 if (!NAV_STATE_QUIT.equals(v.getTag())) {
+                    GoogleAnalyticsUtils.reportFormNavBackward(GoogleAnalyticsFields.LABEL_ARROW);
                     FormEntryActivity.this.showPreviousView(true);
                 } else {
                     FormEntryActivity.this.triggerUserQuitInput();
```

Comment:
I'd actually love to know whether user's are using this v. 'back' v. " < Form Name Here" to exit forms, actually, can we record this too?